base/docs/helpdb/Base.jl

```diff
@@ -784,12 +784,12 @@ Equivalent to `stat(file).mode`
 filemode
 
 """
-    print_joined(io, items, delim, [last])
+    join(io, items, delim, [last])
```

Comment:
removing square brackets around `last` or not? should be consistent between here and the rst docstring at https://github.com/JuliaLang/julia/pull/16603/files#diff-5024f6efcbbb7500d181c34103c2576a